api/src/controllers/cases.ts

```diff
@@ -63,3 +86,20 @@ export const getCasesByCaseId = async (req: Request, res: Response, next: NextFu
     next(err)
   }
 }
+
+export const updateCaseWithNewUser = async (req: Request, res: Response, next: NextFunction) => {
+  const caseId = parseInt(req.params.id)
+  const user: any = req.user!
+  const userId = Number(user.id)
+  const { details } = req.body
+  try {
+    await Case.update({
+      userId,
+    }, { where: { id: caseId } })
+    req.body = { subject: 'Transfer', details }
```

Comment:
So, a few things..

1) Perhaps..
```suggestion
    req.body.subject = 'Transfer'
```

2) The _event_ ought to be saved first. Consider the scenario where this fails halfway. If the user id changes without the corresponding event we would have had no way to prove that this was indeed the user that we planned to assign it to. Having the event saved first allows us to recover state more easily. Better yet, do this within a transaction.

3) .. I'm not even sure whether it's the "right" thing to delegate the event saving to another express controller like that (see 2)